mountpoint-s3/src/fs.rs

```diff
@@ -107,6 +107,42 @@ pub struct DirectoryEntry {
     lookup: LookedUp,
 }
 
+#[derive(Debug)]
+/// Reply to a 'statfs' call
+pub struct StatFs {
+    /// Total number of blocks
+    pub total_blocks: u64,
+    /// Number of free blocks
+    pub free_blocks: u64,
+    /// Number of free blocks available to unprivileged user
+    pub available_blocks: u64,
+    /// Number of inodes in file system
+    pub total_inodes: u64,
+    /// Available inodes
+    pub free_inodes: u64,
+    /// Optimal transfer block size
+    pub block_size: u32,
+    /// Maximum name length
+    pub maximum_name_length: u32,
+    /// Fragement size
+    pub fragment_size: u32,
+}
+
+impl Default for StatFs {
+    fn default() -> Self {
+        Self {
+            total_blocks: 0,
+            free_blocks: 0,
+            available_blocks: 0,
+            total_inodes: 0,
+            free_inodes: 0,
+            block_size: 512,
+            maximum_name_length: 255,
+            fragment_size: 0,
+        }
+    }
+}
```

Comment:
Can you add a comment saying that these defaults were copied over from Fuser?

Reply:
They were originally from here:

https://github.com/cberner/fuser/blob/e18bd9bf9071ecd8be62993726e06ff11d6ec709/src/lib.rs#L695-L698